### ISSUE TITLE
Fix Shortcuts for mac

### ISSUE THREAD

**Issue by ThinkDigitalSoftware** [SolarLune/masterplan#13]
On mac, the new shortcut is _Cmd + n_, but it's currently set to _Ctrl + n_

**Comment by SolarLune** [comment]
Yo! Can you be more specific? Are you talking about the shortcut for sliding Tasks around?

**Comment by ThinkDigitalSoftware** [comment]
The shortcut for creating a new task

**Comment by SolarLune** [comment]
I'm not sure I get what you mean - Ctrl+N is the new task shortcut and behaves correctly.

**Comment by ThinkDigitalSoftware** [comment]
On mac, you usually press command + n for new. But in master plan, it's control +N for new

On Feb 25, 2020 11:54 PM, SolarLune <notifications@github.com> wrote:

I'm not sure I get what you mean - Ctrl+N is the new task shortcut and behaves correctly.

**Comment by SolarLune** [comment]
OK, I wasn't sure if that's what you meant or not. I'll have to think about how best to approach this one, as that would mean a different key for Mac as a platform specifically for any shortcut that uses Control.

**Comment by ThinkDigitalSoftware** [comment]
Is that complicated to implement?

On Feb 26, 2020 12:40 AM, SolarLune <notifications@github.com> wrote:

OK, I wasn't sure if that's what you meant or not. I'll have to think about how best to approach this one, as that would mean a different key for Mac as a platform specifically for any shortcut that uses Control.

**Comment by SolarLune** [comment]
Not really, but it isn't necessarily trivial, as I need to change all shortcuts that use Ctrl to use the Super key when running on Mac specifically. That's an additional layer of redirection that I'll have to add to the existing shortcut system.

**Comment by ThinkDigitalSoftware** [comment]
Ah ok

On Feb 26, 2020 8:59 AM, SolarLune <notifications@github.com> wrote:

Not really, but it isn't necessarily trivial, as I need to change all shortcuts that use Ctrl to use the Super key when running on Mac specifically. That's an additional layer of redirection that I'll have to add to the existing shortcut system.

**Comment by SolarLune** [comment]
So this should be resolved as of the linked commit; I added the ability to use both Control as well as the Super keys for shortcuts.

**Comment by ThinkDigitalSoftware** [comment]
Awesome! Thanks

On Mar 5, 2020 7:59 PM, SolarLune <notifications@github.com> wrote:

So this should be resolved as of the linked commit; I added the ability to use both Control as well as the Super keys for shortcuts.
